taichi/transforms/loop_vectorize.cpp

```diff
@@ -120,22 +120,25 @@ class LoopVectorize : public IRVisitor {
     auto old_vectorize = for_stmt->vectorize;
     if (for_stmt->vectorize != 1)
       vectorize = for_stmt->vectorize;
-    loop_var = for_stmt->loop_var;
+    // TODO: RangeForStmt::loop_var is deprecated
+    // loop_var = for_stmt->loop_var;
```

Comment:
Remove?

Reply:
`loop_vectorize.cpp` is also for vectorization...

Reply:
I mean specifically L124..